README.md

```diff
@@ -85,6 +85,12 @@ An alternative to the Grunt build is provided via the `util.sh` shell script.
 1.  `mkdir google_closure_templates; cd google_closure_templates`
```

Comment:
IIRC when we were chatting you needed these instructions still because of soyutils_usegoog.js, am I remembering correctly?  If so, are the files distributed here in the Closure repo sufficient to remove the need for these steps?

https://github.com/google/closure-templates/tree/master/javascript